rust/signed_doc/schemas/proposal_submission_action.schema.json

```diff
@@ -0,0 +1 @@
+./../../../specs/signed_docs/docs/payload_schemas/proposal_submission_action.schema.json
```

Comment:
Instead of a symlink, it would be a lot cleaner if the rust code just used the file directly, and the build replicated the directory structure so the paths didn't change.